app/templates/layout.html

```diff
@@ -3,6 +3,7 @@
     <head>
         <title>{% block title %}{% endblock %}</title>
         {% block imports %}{% endblock %}
+        {% block scripts %}{% endblock %}
```

Comment:
This should go at the very end of the `<body>`. If you do this, you won't need `window.onload` to initialize your script.